.github/workflows/kind-conformance.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
 
         # Map between K8s and KinD versions.
         # This is attempting to make it a bit clearer what's being tested.
-        # See: https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1
+        # See: https://github.com/kubernetes-sigs/kind/releases/tag/v0.12.0
```

Comment:
We should probably update the knative version used for conformance tests. I see e2e is getting updated, conformance tests likely still using 1.1.0 (from Makefile)